apps/emqx/src/emqx_schema.erl

```diff
@@ -523,7 +524,16 @@ fields("mqtt") ->
                 number(),
                 #{
                     default => 0.75,
-                    desc => ?DESC(mqtt_keepalive_backoff)
+                    deprecated => {since, "5.0.25"}
+                }
+            )},
+        {"keepalive_multiplier",
+            sc(
+                number(),
+                #{
+                    default => 1.5,
```

Comment:
we need a converter at the `"mqtt"` type field (parent struct) to convert `keepalive_backoff` to `keepalive_multipiler`.

Reply:
done